### PR TITLE
Fix dependabot.yaml

### PR DESCRIPTION
**Pull request by lubronzhan**
## What this PR does / why we need it

Fix dependabot since pipeline is failing

## Details for the Release Notes (PLEASE PROVIDE)

```release-note

```

## Which issue(s) this PR fixes

Fixes: #https://github.com/vmware-tanzu/community-edition/issues/3372

### CODE REVIEW

**Comment by lubronzhan** [comment]
I used this fix in CPI related PR, the pipeline passed. Then I removed this from that PR since it's unrelated to CPI

File: .github/dependabot.yml [hunk 1]
```diff
@@ -42,13 +42,6 @@ updates:
     ignore:
       - dependency-name: "tanzu-framework"
 
-  - package-ecosystem: "gomod"
```
Comment (stmcginnis) [inline .github/dependabot.yml:45]: This one shouldn't be removed as the `schedule` configuration is fine.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -140,10 +133,6 @@ updates:
...
-    directory: "addons/packages/harbor/2.2.3/test"
-    schedule:
```
Comment (stmcginnis) [inline .github/dependabot.yml:145]: Rather than removing, this should actually have a schedule value set so it doesn't error on `none`.

**Review by stmcginnis — CHANGES_REQUESTED** [review]
I don't think we want to remove these packages from the configuration.

File: .github/dependabot.yml (hunk 1 above)
Comment (lubronzhan) [reply .github/dependabot.yml:45]: Hi @stmcginnis did you check the failure of the dependabot pipeline? This line is duplicate of this https://github.com/vmware-tanzu/community-edition/blob/0a24ed9bace713bce0e8b6031da1a1009dd3b29f/.github/dependabot.yml#L10-L15

File: .github/dependabot.yml (hunk 2 above)
Comment (lubronzhan) [reply .github/dependabot.yml:145]: Hi @stmcginnis , this package is already  defined here https://github.com/vmware-tanzu/community-edition/blob/main/.github/dependabot.yml#L136-L141
Probably a wrong rebase result

Comment (stmcginnis) [reply .github/dependabot.yml:145]: That one is for harbor 2.3.3, but this one is for 2.2.3. So I think we still want this, just with an actual schedule defined. Looks like maybe a copy/paste problem.

File: .github/dependabot.yml (hunk 1 above)
Comment (stmcginnis) [reply .github/dependabot.yml:45]: The pipeline error I saw was that "schedule" could not be none, so I think the main issue is the one below. But yeah, line 10 already has this one, so +1 to removing it then.

File: .github/dependabot.yml (hunk 2 above)
Comment (lubronzhan) [reply .github/dependabot.yml:145]: Thanks. I didn't notice the version. Let me add the same schedule here

**Review by stmcginnis — APPROVED** [review]
Thanks @lubronzhan!

**Pull request merged**